docs/en-US/component/popover.md

```diff
@@ -80,7 +80,7 @@ popover/directive-usage
 | placement                 | popover placement                                                                                                                                                        | string          | top/top-start/top-end/bottom/bottom-start/bottom-end/left/left-start/left-end/right/right-start/right-end | bottom                                                                     |
 | disabled                  | whether Popover is disabled                                                                                                                                              | boolean         | —                                                                                                         | false                                                                      |
 | visible / v-model:visible | whether popover is visible                                                                                                                                               | Boolean         | —                                                                                                         | false                                                                      |
-| offset                    | popover offset                                                                                                                                                           | number          | —                                                                                                         | 0                                                                          |
+| offset                    | popover offset                                                                                                                                                           | number          | —                                                                                                         | 12                                                                         |
```

Comment:
It seems that the default value of the `offset` property of the `popover` component is `undefined`, while the default value of the corresponding property of the `el-tooltip` component used internally is `12`. Perhaps we could add this explanation to make it clearer?

Reply:
> It seems that the default value of the `offset` property of the `popover` component is `undefined`, while the default value of the corresponding property of the `el-tooltip` component used internally is `12`. Perhaps we could add this explanation to make it clearer?

Agree! I also discovered another error in the document, done!